.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+^^ Provide a general summary of the issue in the title above. ^^ 
+
+**Description**
+Describe the problem you're encountering.
+TIP: Do NOT share sensitive information, whether personal, proprietary, or otherwise! 
+
+**Expected Behavior**
+Tell us what you expected to happen.
+
+**[Troubleshooting](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) or [NR Diag](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) results**
+Provide any other relevant log data.
+TIP:  Scrub logs and diagnostic information for sensitive information 
+
+**Steps to Reproduce**
+Please be as specific as possible.
+TIP:  Link a sample application that demonstrates the issue. 
+
+**Your Environment**
+Include as many relevant details about your environment as possible including the running version of the ruby agent, the ruby version being used and any other relevant environment information
```

Comment:
Either Ruby or ruby is correct, but for consistency with our other documentation it would be good to capitalize it in both instances in this line.